CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ and this project adheres to
 
 ### Added
 
+- [#4428](https://github.com/firecracker-microvm/firecracker/pull/4428): Added
+  basic ACPI support to Firecracker for x86_64 microVMs. This allows booting
+  ACPI-enabled kernels. Currently, we pass ACPI tables with information about
```

Comment:
`ACPI-enabled kernels` might be confusing since IIUC mptables also allowed booting kernels that had ACPI configs enabled.

Reply:
Re-worded it to mention that we can boot kernels without MPTable support.